collections/_article/godot-showcase-usagi-shima.md

```diff
@@ -0,0 +1,67 @@
+
+![](/assets/showcase/usagi-shima-4.webp)
+
+## What were some of the biggest challenges you faced during the development process, and how did you overcome them?
+
+By far the biggest challenge for me was not even making the game itself, but integrating the game with each platform’s ecosystem. That means integration with Admob, Google Play Game Services, Google Play Billing, iCloud, Game Center, etc. I was juggling this mixture of third party plugins and SDKs, and that honestly makes me a bit nervous about potential difficulties ahead maintaining the game long term. 
+
+Keeping up with Google Play’s constantly changing requirements and policies was also challenging. Sometimes I had to keep my eyes on open issues on Godot’s Github page to see when a required Android change would land in the next release so that I could ensure my app would remain compliant with Google Play policies. A similar issue arose when Google Play started to enforce a new minimum version of the Google Play Billing library, but the latest official release from the Godot Google Play Billing plugin was still behind.
+
+Indeed, having been part of the Linux community in my past job, I completely understand the growing pains of any open source community. Maintainers are often overworked and stretched super thin. But at the same time, what’s totally awesome about open source is how other users can step up and help each other and contribute to the project. That was the case with the Google Play Billing plugin where one user kindly created a release candidate that fixed the minimum library version requirement. I’m still using that rc and am hoping the repo gets an official update at some point 😉
+
+If I had a Godot wishlist, it would be a first-class plugin for Google Play Game Services, and maybe even a dedicated maintainer for all things Android/Google Play plugin related. Although the community has done a fantastic job creating plugins, I worry that third-party plugins will stop being maintained and as mobile devs we’ll thus be forced to switch to yet another plugin, that may or may not be actively maintained in the future.
+
+
+## How did you discover Godot? What made you pick it for your project?
+
+I was already coming from an open source background when I worked on Linux, so it felt very natural for me to pick an open source game engine. The open source aspect is a big draw for me as I’m used to looking through source code and discussions on Github if I get stuck on an issue. In addition I’m pretty sure it was recommended on /r/gamedev for being a solid, intuitive engine for newcomers as well. I was also able to hit the ground running with Godot just after just watching [Heartbeast’s beginner RPG series](https://www.youtube.com/watch?v=mAbG8Oi-SvQ). I just love how intuitive the node structure is and Godot’s ease-of-use - it was so easy to prototype things with GDScript!
```

Comment:
```suggestion
I was already coming from an open source background when I worked on Linux, so it felt very natural for me to pick an open source game engine. The open source aspect is a big draw for me as I’m used to looking through source code and discussions on Github if I get stuck on an issue. In addition I’m pretty sure it was recommended on [/r/gamedev](https://www.reddit.com/r/gamedev/) for being a solid, intuitive engine for newcomers as well. I was also able to hit the ground running with Godot just after just watching [Heartbeast’s beginner RPG series](https://www.youtube.com/watch?v=mAbG8Oi-SvQ). I just love how intuitive the node structure is and Godot’s ease-of-use - it was so easy to prototype things with GDScript!
```

Reply:
I'm also not sure about the "just after just"...

Reply:
yeah, that one seems like a typo, so feel free to submit that edit as well 🙏

Reply:
Thing is I'm not sure what she meant. It could be "just after" as in very soon after watching the videos, or it could be "after just" as in after only watching the videos and not needing any other resources. Or it could indeed be "just after just" as a combination of both of those!